build/build.proj

```diff
@@ -179,8 +186,27 @@
   -->
   <Target Name="BuildVS14" Condition=" '$(IsXPlat)' != 'true' ">
     <Message Text="Building for VS14" Importance="high" />
+    <CallTarget Targets="BuildVS14NoVSIX;BuildVS14VSIX"/>
```

Comment:
Wire this up using aftertargets instead as it leads to a more debuggable target graph